app/handlers/ErrorHandler.scala

```diff
@@ -32,12 +32,16 @@ class ErrorHandler @Inject()(
                               val messagesApi: MessagesApi,
                               view: ErrorTemplate,
                               noDataEnteredView: NoDataEnteredErrorView,
+                              pageNotFoundView: PageNotFoundErrorView,
                               config: FrontendAppConfig
                             ) extends FrontendErrorHandler with I18nSupport {
 
   override def standardErrorTemplate(pageTitle: String, heading: String, message: String)(implicit rh: Request[_]): Html =
     view(pageTitle, heading, message)
 
+  override def notFoundTemplate(implicit request: Request[_]): Html =
+    pageNotFoundView(config.contactHmrcURL)(Request(request, ""), request2Messages(request))
```

Comment:
Do we need to explicitly pass the request and messages here? I could be wrong but I think you can just do `pageNotFoundView(config.contactHmrcURL)`